index.html

```diff
@@ -216,6 +216,57 @@ <h3 class="skill-heading">Skills</h3>
           </ul>
         </div>
       </section>
+      <footer>
+        <div class="container container-information">
+           <div class="top-footer">
+              <h2 class="footer-heading">Get Started</h2>
+              <p class="last-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+              <form>
+                <ul>
+                  <li class="input-item">
+                    <label class="label1">Your name</label>
+                    <input class="input2" type="text" id="name" name="user-name" required maxlength="30">
+                  </li>
+                  <li class="input-item">
+                    <label class="label1">Your email</label>
+                    <input class="input2" type="email" id="mail" name="user-email" required>
+                  </li>
+                  <li class="input-item">
+                    <label>message</label>
+                    <textarea id="textarea1" placeholder="Enter your message here..." required maxlength="500"></textarea>
+                  </li>
+                  <li class="input-item">
+                    <button class="cta-btn-last">Start collaboration</button>
+                  </li>
+                </ul>
+              </form>
```

Comment:
- [ ] kindly to to follow the Figma design when it comes to padding and margins for the form elements and the border styles as well 
![image](https://user-images.githubusercontent.com/89282221/187510846-6f4f5355-c5f2-40d6-9752-892777821362.png)